apps/thunderhub/docker-compose.yml

```diff
@@ -2,8 +2,7 @@ version: "3.7"
 
 services:
   web:
-    image: apotdevin/thunderhub:v0.12.31@sha256:f86341cb1987a3907f7227ccd4c00dead68d372e464b111414b8427228062276
-    user: "1000:1000"
+    image: apotdevin/thunderhub:v0.13.7@sha256:92f8b01e666338a1b0742b8707a1902f0bf835a54d528db4cf255726f55786c5
```

Comment:
With https://github.com/apotdevin/thunderhub/pull/434 merged, it will no longer be necessary to remove `user: "1000:1000"` as the Thunderhub image will work nicely as a non-root user.